clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -82,67 +82,65 @@ export class CopybookDownloadService implements vscode.Disposable {
     return action === UNLOCK_DOWNLOAD_QUEUE_MSG;
   }
 
-  private static isInvalidCredentials(e: any) {
+  public isInvalidCredentials(e: any) {
     return e?.mDetails?.errorCode === 401;
   }
 
-  private static async fetchCopybook(
+  public async fetchCopybook(
     dataset: string,
     copybookProfile: CopybookProfile,
     isUSS: boolean,
+    storagePath: string,
```

Comment:
Now that these functions are no longer static, why aren't they using the `this.storagePath`.

Having static with extra parameter would make sense, having non-static and rely on the member variable would make sense, doing both does not make any sense to me.
That also applies to all the other originally static functions.

Reply:
should be resolved at : 8ce0f8515a9a2385de1d9abbff255fdab177093a